debug/src/debug.js

```diff
@@ -67,7 +67,21 @@ export function initDebug() {
 			}
 		}
 
-		oldCatchError(error, vnode, oldVNode);
+		try {
+			oldCatchError(error, vnode, oldVNode);
+
+			// when an error was handled by an ErrorBoundary we will nontheless emit an error
+			// event on the window object. This is to make up for react compatibility in dev mode
+			// and thus make the Next.js dev overlay work.
+			if (typeof error.then != 'function') {
+				setTimeout(() => {
+					console.log('re-throw', error);
```

Comment:
FYI: This was probable left in by accident.

Reply:
```suggestion
```